CONTRIBUTING.md

```diff
@@ -320,15 +320,15 @@ any other valid frontmatter field | see the frontmatter definitions.
 
 Generated files are managed using the `make.py` Python script located in the root of this repository. To use it, first install the required Python packages via the following command:
 
-    pip install --requirement tools/generation/requirements.txt
+    pip install pipenv && pipenv install
```

Comment:
Possibly it'll be useful to add a note that to use `pipenv`, Python should be in PATH

Reply:
If this patch is desirable to the maintainers of this project I'd be happy to add some prose to that effect.